.github/workflows/e2e_tests.yml

```diff
@@ -0,0 +1,101 @@
+name: E2E Tests
+
+on:
+  deployment_status:
+
+jobs:
+  e2e-tests:
+    if: |
+      github.event_name == 'deployment_status' &&
+      github.event.deployment_status.state == 'success' &&
+      (
+        (
+          contains(github.event.deployment.environment, 'Production') &&
+          contains(github.event.deployment_status.target_url, 'liam-erd-web')
+        ) ||
+        (
+          contains(github.event.deployment.environment, 'Preview') &&
+          !contains(github.event.deployment_status.target_url, 'liam-erd-sample') &&
```

Comment:
It is better to E2E test `liam-erd-sample` as well, what do you think?

Reply:
That was true.🙏
I have also made the test run on ``erd-sample``.
However, let me add this in the next pull request because VRT（mobile） was always in a state to fail.💭
https://github.com/liam-hq/liam/actions/runs/13916160365/job/38939497913
I give up adding to this PR.

Reply:
This is difficult... thank you!